# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1247166274 14400
#      Thu Jul 09 15:04:34 2009 -0400
# Node ID 69d05d1a63b990654aa6f3d92916aabf2ac4d403
# Parent  f284e2a26ff9ba22cfee39633f0cc97e303c4896
configure.in: don't use system strftime on MinGW systems

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2009-07-09  John W. Eaton  <jwe@octave.org>
+
+	* configure.in: Don't use system strftime on MinGW systems.
+
 2009-06-29  Jaroslav Hajek  <highegg@gmail.com>
 
 	* NEWS: Correct info.
 	* NEWS: Update.
 
 2009-06-26  Michael Goffioul  <michael.goffioul@gmail.com>
 
 	* aclocal.m4: Add pkg.m4 macros.
diff --git a/configure.in b/configure.in
--- a/configure.in
+++ b/configure.in
@@ -1649,18 +1649,18 @@ case "$canonical_host_type" in
     [AC_MSG_RESULT(no)
      HAVE_MKSTEMPS=no
      LIBS="$save_LIBS"
     ])
    ;;
 esac
 
 case "$canonical_host_type" in
-  *-*-msdosmsvc)
-    ## The %T format specifier for strftime is reportedly broken,
+  *-*-msdosmsvc | *-*-mingw*)
+    ## The %T and %e format specifiers for strftime are not implemented
     ## so use our version.  We could use an actual configure test
     ## for this.
   ;;
   *)
     AC_CHECK_FUNCS(strftime)
   ;;
 esac
 
